src/test/java/edu/ucsb/cs156/happiercows/controllers/CommonsControllerTests.java

```diff
@@ -49,394 +48,394 @@
 @AutoConfigureDataJpa
 public class CommonsControllerTests extends ControllerTestCase {
 
-  @MockBean
-  UserCommonsRepository userCommonsRepository;
-
-  @MockBean
-  UserRepository userRepository;
-
-  @MockBean
-  CommonsRepository commonsRepository;
-
-  @Autowired
-  private ObjectMapper objectMapper;
-
-  @WithMockUser(roles = { "ADMIN" })
-  @Test
-  public void createCommonsTest() throws Exception {
-    LocalDateTime someTime = LocalDateTime.parse("2022-03-05T15:50:10");
-    LocalDateTime someOtherTime = LocalDateTime.parse("2022-04-20T15:50:10");
-
-    Commons commons = Commons.builder()
-        .name("Jackson's Commons")
-        .cowPrice(500.99)
-        .milkPrice(8.99)
-        .startingBalance(1020.10)
-        .startingDate(someTime)
-        .endingDate(someOtherTime)
-        .degradationRate(50.0)
-        .showLeaderboard(false)
-        .build();
-
-    CreateCommonsParams parameters = CreateCommonsParams.builder()
-        .name("Jackson's Commons")
-        .cowPrice(500.99)
-        .milkPrice(8.99)
-        .startingBalance(1020.10)
-        .startingDate(someTime)
-        .endingDate(someOtherTime)
-        .degradationRate(50.0)
-        .showLeaderboard(false)
-        .build();
-
-    String requestBody = objectMapper.writeValueAsString(parameters);
-    String expectedResponse = objectMapper.writeValueAsString(commons);
-
-    when(commonsRepository.save(commons))
-        .thenReturn(commons);
-
-    MvcResult response = mockMvc
-        .perform(post("/api/commons/new").with(csrf())
-            .contentType(MediaType.APPLICATION_JSON)
-            .characterEncoding("utf-8")
-            .content(requestBody))
-        .andExpect(status().isOk())
-        .andReturn();
-
-    verify(commonsRepository, times(1)).save(commons);
-
-    String actualResponse = response.getResponse().getContentAsString();
-    assertEquals(expectedResponse, actualResponse);
-  }
+    @MockBean
+    UserCommonsRepository userCommonsRepository;
+
+    @MockBean
```

Comment:
If this is only whitespace/formatting changes, PLEASE DON'T DO THIS.

It makes your PR super hard to code review, because we have to look at every blessed line to make sure you didn't delete or add something.

Fixing formatting should be done in a separate PR that ONLY focuses on fixing formatting.